librecad/src/actions/rs_actionpolylinesegment.cpp

```diff
@@ -54,9 +54,9 @@ void RS_ActionPolylineSegment::init(int status) {
 	if (container->countSelected(true, entityType)) {
 		//find a selected entity
 		//TODO, find a better starting point
-		for (RS_Entity* e = container->firstEntity(RS2::ResolveAllButInserts);
+		for (RS_Entity* e = container->firstEntity(RS2::ResolveNone);
```

Comment:
polylines are of lines/arcs (wondering why not elliptical arcs as well), but doesn't accept, for example, another polyline as a member.

Maybe, some day, there can be a poly-polyline, but it's not attractive either.